Dockerfile.ci

```diff
@@ -21,16 +21,33 @@ RUN INSTALL_PKGS="nodejs nodejs-nodemon npm findutils tar" && \
 WORKDIR ${HOME}
 
 ENV HUSKY=0
-COPY package*.json .
-RUN npm ci
+COPY web/package*.json ${HOME}/web/
+COPY Makefile Makefile
+RUN make install-frontend-ci
 
-COPY . ${HOME}
-RUN npm run build
+COPY web/ ${HOME}/web
+RUN make build-frontend
 
-FROM registry.redhat.io/ubi8/nginx-120:1-44
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18 as go-builder
+
+WORKDIR /opt/app-root
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY vendor/ vendor/
```

Comment:
We should remove the vendor folder as downstream no longer using vendoring.  You may need to add a step[](https://github.com/openshift/cluster-logging-operator/blob/master/Dockerfile#L19) to download the deps before building

Reply:
Ref: https://github.com/openshift/cluster-logging-operator/blob/master/Dockerfile#L19